client/blocks/reader-post-card/featured-asset.jsx

```diff
@@ -2,35 +2,55 @@
  * External Dependencies
  */
 import React from 'react';
-import { head, filter, get } from 'lodash';
+import { head, filter } from 'lodash';
 
 /**
  * Internal Dependencies
  */
 import FeaturedVideo from './featured-video';
 import FeaturedImage from './featured-image';
 
+// TODO delete canonical image logic from rules?
```

Comment:
I was thinking that picking the feature might just be a new rule, like pickCanonicalImage. Then it would ignore the canonical image entirely and this can go away.